### PR TITLE
Some small random tgui-next uis

### PR DESCRIPTION
**Pull request by actioninja**
## About The Pull Request

Holodeck:
image

Engraved messages:
image

DNA Vault:
image

Spawners:
image

## Why It's Good For The Game

good ui yes

## Changelog
:cl:
add: DNA vault, holodeck, spawners, and engraved message tgui-next
/:cl:

### CODE REVIEW

**Comment by actioninja** [comment]
Something is broken with build checking, I confirmed 3 times that this in fact the proper build for this pr.

**Comment by Tharcoonvagh** [comment]
Safe ui too please

**Comment by stylemistake** [comment]
@actioninja my local build differs too, so the best guess is that somehow your local `node_modules` folder got fucked up, and you need to clean build.

```
bin/tgui --clean
bin/tgui
```

**Review by stylemistake — APPROVED** [review]

File: code/game/objects/structures/ghost_role_spawners.dm [hunk 1]
```diff
@@ -322,8 +332,10 @@
 	name = "hotel security sleeper"
 	mob_name = "hotel security member"
 	outfit = /datum/outfit/hotelstaff/security
-	flavour_text = "<span class='big bold'>You are a peacekeeper</span><b> assigned to this hotel to protect the interests of the company while keeping the peace between \
-		guests and the staff. Do <font size=6>NOT</font> leave the hotel, as that is grounds for contract termination.</b>"
+	short_desc = "You are a peacekeeper."
+	flavour_text = "You have been assigned to this hotel to protect the interests of the company while keeping the peace between \
+		guests and the staff."
+	important_info = "Do NOT leave the hotel, as that is grounds for contract termination.</b>"
```
Comment (AnturK) [inline code/game/objects/structures/ghost_role_spawners.dm]: `</b>`

**Review by AnturK — CHANGES_REQUESTED** [review]

File: code/game/objects/structures/ghost_role_spawners.dm (hunk 1 above)
Comment (actioninja) [reply code/game/objects/structures/ghost_role_spawners.dm]: </b>

**Pull request merged**